CMakeLists.txt

```diff
@@ -116,7 +140,7 @@ if(SECP256K1_ENABLE_MODULE_FROST)
   if(NOT SECP256K1_EXPERIMENTAL)
     message(FATAL_ERROR "FROST module is experimental. Use -DSECP256K1_EXPERIMENTAL=ON to allow.")
   else()
-    add_definitions(-DENABLE_MODULE_FROST=1)
+    add_compile_definitions(-DENABLE_MODULE_FROST=1)
```

Comment:
In other sections of `CMakeLists.txtwhen replacing `add_definitions()` with `add_compile_definitions()`, upstream also got rid of `-D`.

See for example this snippet taken from **v0.3.2**:

https://github.com/bancaditalia/secp256k1-frost/blob/acf5c55ae6a94e5ca847e07def40427547876101/CMakeLists.txt#L56

We could try to do the same in order to be as aligned to upstream as possible.